packages/apps/api-authorizer/src/index.ts

```diff
@@ -89,18 +81,11 @@ export async function lambdaHandler(
       event.pathParameters
     )
   ) {
-    console.log(
-      'Access token [' +
-        accessToken +
-        '] for user [' +
-        JSON.stringify(auth0UserInfo) +
-        '] cannot operate on [' +
-        event.httpMethod +
-        ' ' +
-        event.resource +
-        '] with path parameters [' +
-        event.pathParameters +
-        ']'
+    console.debug(
+      `Access token [${accessToken}] for user [${JSON.stringify(
+        auth0UserInfo
+      )}]` +
+        `cannot operate on [${event.httpMethod} ${event.resource}] with path parameters [${event.pathParameters}]`
```

Comment:
Probably not one to fix now, but we should look at exactly what's in that `auth0UserInfo` variable before we go logging it everywhere.

Reply:
shouldn't be an issue at the debug level, but yes, should be done especially for the other cases

Reply:
Definitely PII: https://github.com/wellcomecollection/identity/blob/45c9badfc2aa8ba7197530b1a64193836a0056fb/packages/shared/auth0-client/src/auth0.ts#L151

Reply:
> shouldn't be an issue at the debug level

Not sure I agree. I don't love the idea that changing the log level suddenly starts putting PII into our logging cluster.

Reply:
True (although I got the [idea](https://wellcome.slack.com/archives/CUA669WHH/p1618396869015100?thread_ts=1618395149.009100&cid=CUA669WHH) from you 😬) - I will return to this in my next PR

Reply:
The person who wrote that message looks very shifty. I wouldn't trust anything they say! 😉

Reply:
Encouraging accidental leaking of PII is the kind of thing a spy might do